aries-site/src/data/wcag/components.json

```
@@ -3003,5 +3003,519 @@
         "status": "not-applicable"
       }
     ]
+  },
+  {
+    "checkbox": [
+      {
+        "rule": "1.1.1",
+        "label": "Non-text content",
+        "status": "passed",
+        "notes": "Tested as part of checkbox + label"
+      },
+      {
+        "rule": "1.2.1",
+        "label": "Audio-only and Video-only (Prerecorded)",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "1.2.2",
+        "label": "Captions (Prerecorded)",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "1.2.3",
+        "label": "Audio Description or Media Alternative (Prerecorded)",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "1.2.4",
+        "label": "Captions (Live)",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "1.2.5",
+        "label": "Audio Description (Prerecorded)",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "1.2.6",
+        "label": "Sign Language (Prerecorded)",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "1.2.7",
+        "label": "Extended Audio Description (Prerecorded)",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "1.2.8",
+        "label": "Media Alternative (Prerecorded)",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "1.2.9",
+        "label": "Audio-only (Live)",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "1.3.1",
+        "label": "Info and Relationships",
+        "status": "passed",
+        "notes": ""
+      },
+      {
+        "rule": "1.3.2",
+        "label": "Meaningful Sequence",
+        "status": "passed",
+        "notes": ""
+      },
+      {
+        "rule": "1.3.3",
+        "label": "Sensory Characteristics",
+        "status": "passed",
+        "notes": ""
+      },
+      {
+        "rule": "1.3.4",
+        "label": "Orientation",
+        "status": "passed",
+        "notes": ""
+      },
+      {
+        "rule": "1.3.5",
+        "label": "Identify Input Purpose",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "1.3.6",
+        "label": "Identify Purpose",
+        "status": "failed",
+        "notes": "Screen reader reads out misleading information"
```

Can we specify that this is related to the indeterminate state of the checkbox?
```suggestion
        "notes": "the screenreader reads out the indeterminate state checkbox as unchecked without additional information. It feels like we are not giving the screenreader user the same information as a sighted user. The indeterminate state is different than the unchecked state."
```